drain3/drain.py

```diff
@@ -280,6 +282,6 @@ def add_log_message(self, content: str):
 
     def get_total_cluster_size(self):
         size = 0
-        for c in self.clusters:
+        for c in self.clusters.values():
```

Comment:
Yes, the get_total_cluster_size no longer reflects the number of processed entries. I don't think that we need it as it is trivial to keep track of such counter by the callers.

Reply:
I think it will be useful to know how many clusters would you have if max_clusters was not bounded - to see how many clusters you have "lost". And its not so trivial to keep track of that by the caller (you need to increment every time the add_log_message() result is `new_cluster`. However, I am not sure it will be the same count necessarily ..

Reply:
There are several metrics we could track here. Previously `get_total_cluster_size` was matched the total number of log entries processed with `add_log_message`. This is what I had in mind when suggesting that callers can easily keep track of it. 

Another interesting metric is the number of clusters. Previously it wasn't exposed directly but one could access it with a simple `len(model.clusters)`. With the changes I made this is still possible with a caveat that this number reflects the number of active/existing clusters and will most likely match `max_clusters` when the model saturates the cache.

I think that the new definition of the `get_total_cluster_size` is the most useful as one can divide it by the number of entries to get a sense of *cluster coverage* in the log stream.

Other metrics can be added as necessary, but I don't have examples of strong use cases now.